File: support_tools/swigwin-3.0.12/Examples/test-suite/go/voidtest_runme.go
package main

import "./voidtest"

func main() {
	voidtest.Globalfunc()
	f := voidtest.NewFoo()
	f.Memberfunc()

	voidtest.FooStaticmemberfunc()

	v1 := voidtest.Vfunc1(f.Swigcptr())
	v2 := voidtest.Vfunc2(f)
	if v1 != v2 {
		panic(0)
	}

	v3 := voidtest.Vfunc3(v1)

	v4 := voidtest.Vfunc1(f.Swigcptr())
	if v4 != v1 {
		panic(0)
	}

	v3.Memberfunc()
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/voidtest_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'voidtest'

Voidtest.globalfunc()
f = Voidtest::Foo.new
f.memberfunc()

Voidtest::Foo.staticmemberfunc()

v1 = Voidtest::vfunc1(f)
v3 =  Voidtest::vfunc3(v1)

v3.memberfunc()

